docker/docker-compose-openapi.yml

```diff
@@ -0,0 +1,51 @@
+version: "3.6"
+
+services:
+  scxa-gradle:
+    image: openjdk:11
+    container_name: scxa-gradle
+    networks:
+      - scxa
+    ports:
+      - "8081:8081"
+    working_dir: /root/project
+    volumes:
+      - root-gradle-wrapper:/root/.gradle/wrapper
+      - root-gradle-caches:/root/.gradle/caches
+      - ..:/root/project
+      - ./packages:/root/.m2
+      - /home/alf/gradle-ro-dep-cache:/gradle-ro-dep-cache
```

Comment:
Still showing your user directory path `/home/alf/gradle-ro-dep-cache`. Would it be possible to make it generic?

Reply:
Oh no! I changed it elsewhere but I forgot here. Well spotted. Done.